package.json

```diff
@@ -24,6 +24,8 @@
     }
   ],
   "dependencies": {
+    "@builder.io/qwik": "^1.2.10",
+    "@builder.io/qwik-city": "^1.2.10",
```

Comment:
```suggestion
```
Is this change intentional? I think is a typo. 🤔

Reply:
These dependencies were not added and I accidentally added them when I was trying to figure out how to integrate the functionality locally.

At the time I was doing pnpm link.dist, to create the symbolic link that was fine, then I had to add the reference of those packages to the new test project and without realizing it I did it in the "qwik" project itself, with the consequent error due to For my part, I have tried to solve it so as not to have problems.

In the main branch link, if you look, that's how it is, so it was my mistake.
https://github.com/BuilderIO/qwik/blob/main/package.json

![image](https://github.com/BuilderIO/qwik/assets/105705996/1a36f663-429f-475a-a056-db7880be1b91)